### DISCUSSION TITLE
Month and Year options aren't showing when clicked inside the calendar in modal

### DISCUSSION THREAD

**Discussion by usmanshakeel662** [twbs/bootstrap#40667] [Q&A]
i've a bootstrap built-in modal that has a date and month and year selection in a calendar when I click on month or year select the options show once then suddenly hide and close button of modal focus/active I don't know what's happening, cuz i also applied stopPropagation
how to solve that. please!

![Screenshot_2](https://github.com/user-attachments/assets/8604ddcc-6b91-4bff-94ca-e5b28549e02e)

**Comment by julien-deramond** [comment]
Hi @usmanshakeel662 
Would it be possible to share some code via CodePen (or any other tools like that) to help the community have the precise context in order to be able to help you?

**Reply by usmanshakeel662** [reply]
Here's the link of code via CodePen :
https://codepen.io/ioknuatk-the-typescripter/pen/OJeXXeN
